00_Core/Scripts/Source/MilkQUEST.psc

```diff
@@ -3030,7 +3030,7 @@ int Function Pain(Actor akActor, int pain)
 		endif
 	endif
 
-	PainCnt += (pain - MaidLevel * 0.025) * (1 - SLA.GetActorArousal(akActor) / 100 * 0.25)
+	PainCnt += (pain - 0.025*MaidLevel) * (1 - SLA.GetActorArousal(akActor) / 100 * 0.25)
```

Comment:
Changed the order of operands because multiplying an integer with a float ~might~ result in an integer value. Multiplying a float with an integer is safe.